rpm/genpatchlist.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+num=0
+# Order patches with zeros in front so rpm applies the patches in the specified order
+zeros=000
+
+while read -r patch ; do
+    echo Patch${zeros}${num}: patches/debian/"$patch"
+    export num=$((num+1))
+    if [ $((num/100)) -eq 1 ] ; then
+        zeros=0
+    elif [ $((num/10)) -eq 1 ] ; then
+        zeros=00
+    fi
+
+    # read patching order from debian quilt file
+done < rpm/patches/debian/series
```

Comment:
I don't see this file listed in spec. All files under rpm/ directory must be listed as a patch or source file to get through CI.